source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -455,6 +456,30 @@ message TransformationAccessChain {
 
 }
 
+message TransformationAddBitInstructionSynonym {
+
+  // A transformation that adds synonyms for bit instructions by evaluating each pair of bits.
+
+  // The bit instruction result id.
```

Comment:
Same (and similar elsewhere - please change the names of the transformation and fuzzer pass classes too).